Doxyfile

```diff
@@ -266,7 +266,7 @@ SEPARATE_MEMBER_PAGES  = NO
 # uses this value to replace tabs by spaces in code fragments.
 # Minimum value: 1, maximum value: 16, default value: 4.
 
-TAB_SIZE               = 4
+TAB_SIZE               = 8
```

Comment:
could you addres this a separate commit ( you can include it in the PR)

Reply:
@arnopo  No problem - done.